### DISCUSSION TITLE
SOC value not shown in home assistant

### DISCUSSION THREAD

**Discussion by hnehnes** [wills106/homeassistant-solax-modbus#383] [Q&A] [answered]
Hi, 

I have a QCells Q.VOLT_HYB-G3-3 and a 12 kWh battery. Many values are displayed, but not the SOC. On the inverter and in the cloud I can see it but not in home assistant. Can you tell me if this value is missing in the modbus communication or how I can add it?

Thank you 
Hendrik

**Comment by wills106** [comment, marked as answer]
The default name should be sensor.solax_battery_capacity

**Reply by hnehnes** [reply]
that worked. thank you very much!
